content/blog/go-embedding-interfaces/index.md

```diff
@@ -0,0 +1,373 @@
+---
+{
+    title: 'Embedding Interfaces in Go',
+    published: '2024-03-04',
+    authors: ['rusher2004'],
+    tags: ['go', 'testing'],
+    description: ""
```

Comment:
Hey @rusher2004 - let's add a description here (no more than 160 chars please!) and then we can ship :)

Reply:
159 characters 😎